pokerbot.js

```diff
@@ -131,6 +137,23 @@ pokerbot.root = function (req, res, next) {
       pokerbot.pokerDataModel[jiraId].channelId['id'] = channel.id
       pokerbot.pokerDataModel[jiraId].channelId['name'] = channel.name
       pokerbot.pokerDataModel[jiraId].channelId['members'] = channel.members.length
+      pokerbot.pokerDataModel[jiraId].channelId['membersList'] = []
+      pokerbot.pokerDataModel[jiraId].channelId['membersList'] = channel.members
+      let memberId
+      let miisingMemberArray = []
+      for (let index = 0; index < channel.members.length; index++) {
+        memberId = channel.members[index]
+        if (!pokerbot.allUsersInTeam.hasOwnProperty(memberId)) {
+          miisingMemberArray.push(memberId)
+        }
+      }
```

Comment:
`for` loops are usually more prone to error rather than using a lodash function. For example, these six lines could be:
```
let missingMembers = _.filter(channel.members, (item) => {
  return pokerbot.allUsersInTeam.hasOwnProperty(item)
})
```